### PR TITLE
feat(twap-orders): refactor of quote (1)

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Refactors:
- useQuoteParams
- useGetQuote
- useIsQuoteLoading

### CODE REVIEW

File: src/modules/tradeQuote/hooks/useQuoteParams.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-// TODO: probably also can be unified for each trade widget
 export function useQuoteParams() {
   const { chainId, account } = useWalletInfo()
-  const { inputCurrency, outputCurrency } = useAtomValue(advancedOrdersDerivedStateAtom)
-  const { typedValue } = useAtomValue(advancedOrdersAtom)
+  const { state } = useDerivedTradeState()
+
+  const inputCurrency = state?.inputCurrency
+  const outputCurrency = state?.outputCurrency
+  const typedValue = state?.typedValue
```
[the remark this answers is not in the document]
Comment (shoom3301) [reply src/modules/tradeQuote/hooks/useQuoteParams.ts:14]: Yes, so it should be:
`state.orderKind === OrderKind.SELL ? state.inputCurrencyAmount : state.outputCurrencyAmount`

[the remark this answers is not in the document]
Comment (shoom3301) [reply src/modules/tradeQuote/hooks/useQuoteParams.ts:14]: p.s. for limit and advanced orders `orderKind` should always be `SELL`

[the remark this answers is not in the document]
Comment (nenadV91) [reply src/modules/tradeQuote/hooks/useQuoteParams.ts:14]: Hmm, I thought LIMIT orders can also be buy 
Screenshot 2023-05-23 at 16 44 36

[the remark this answers is not in the document]
Comment (nenadV91) [reply src/modules/tradeQuote/hooks/useQuoteParams.ts]: Addressed this in a separate PR that is merged in this one already

**Pull request merged**